frb_example/integrate_third_party/rust/src/third_party/web_audio_api/node/mod.rs

```diff
@@ -20,6 +20,11 @@ impl AnalyserNode {
 impl AudioBufferSourceNode {
     #[frb(ignore)]
     pub fn buffer() {}
+
+    // `setbuffer()` is overrided by `setAudioBuffer()` (the argument is cloned so that dart continues to be able to access it).
+    #[frb(ignore)]
+    pub fn set_buffer() {}
+
```

Comment:
...and then we do not need to ignore this (since frb_override will auto override it)